style.css

```diff
@@ -1,3 +1,16 @@
 body {
   background-color: aqua;
-}
\ No newline at end of file
+}
+body {
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+      'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+      sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```

Comment:
kindly make sure to remove the unwanted font-family in the` body `selector. use of extra `font-family` is not required in this project.